docs/software/index.md

```diff
@@ -9,36 +9,94 @@
 [singularity-jobs]: ../runjobs/scheduled-jobs/container-jobs.md
 [software-stacks]: ../runjobs/lumi_env/softwarestacks.md
 [module-env]: ../runjobs/lumi_env/Lmod_modules.md
+[software-library]: https://lumi-supercomputer.github.io/LUMI-EasyBuild-docs
 
 ---
-Here you find guidance on ways to install software on LUMI as well as guides on
-using some specific software packages on LUMI. If you are looking for ways to
+On this page you find guidance on ways to install additional software on LUMI. 
+A full overview of software installed in the LUMI software
+stacks or available as a build recipe for the 
+[EasyBuild][easybuild] package manager is available in the
+[LUMI software library](https://lumi-supercomputer.github.io/LUMI-EasyBuild-docs/).
+If you are looking for ways to
 optimize your software for use on LUMI, consult the [developing
 section][developing-overview] instead.
 
 ---
 
-You have two options for bringing your scientific software to LUMI:
-
-1. Install your scientific software in your home or project directory. For this
-   we recommend using the [EasyBuild][easybuild] package manager as it is used
-   to manage the [central software stack][software-stacks] on LUMI and makes it
-   easy to install additional software that extends it. Alternatively, you may
-   use [Spack][spack] or the [container wrapper][contwrapper].
-2. Bring a [Singularity/Apptainer software container][singularity-container]
-   and run it using the [Singularity container runtime][singularity-jobs]
-   provided by LUMI.
-
-Consult the [module environment page][module-env] for instructions on
-identifying the already available software on LUMI. Furthermore, you may want to
-consult the [EasyBuild][easybuild] and [Spack][spack] pages for information
-about applications for which we already provide recipes that make them very
-easy to install yourself.
-
 !!! warning
 
     The home and project directories reside on the Lustre based parallel
     file system on [LUMI-P][lumi-p] which does not perform well with
     installations of software containing a lot of small files, e.g. Python or
     R environments installed via Conda or pip. For such software a container
-    based approach must be used.
+    based approach must be used. See below for the [container wrapper approach][contwrapper]
```

Comment:
As a bullet list is used rather than separate sections, linking to places below is not always possible unless native HTML is used in the page to create the anchors.